packages/appstore/package.json

```diff
@@ -37,6 +37,7 @@
         "@deriv/trader": "^3.8.0",
```

Comment:
Not related to this PR but can we remove this duplicate entry of `"@deriv/hooks": "^1.0.0"`

Reply:
@nijil-deriv will do